modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -239,6 +239,7 @@ export interface AddressesOptions extends PaginationOptions {
   segwit?: boolean;
   chains?: number[];
   includeBalances?: boolean;
+  includeTokens?: boolean;
```

Comment:
Is there documentation for this on the api docs?

Reply:
Not yet, I can mark this as draft for now and add those in. Thanks :)

Reply:
Re-opening this PR as this has been added to the API docs. Thanks
https://github.com/BitGo/bitgo-microservices/pull/21866